pkg/nexus/common/utils/helper-functions_test.go

```diff
@@ -15,7 +15,7 @@ type HelperSuite struct {
 func (helperSuite *HelperSuite) TestGetEnvironmentHost() {
 	hostEnv := GetEnvironmentHost()
 
-	helperSuite.Equal("localhost", hostEnv)
+	helperSuite.Equal("host.docker.internal", hostEnv)
```

Comment:
please dont hardcode "host.docker.internal"

Reply:
This is a test not a function

Reply:
And on linux it works too